kie-wb-tests/kie-wb-tests-rest/src/test/java/org/kie/wb/test/rest/RestTestBase.java

```diff
@@ -28,9 +33,11 @@
 
 public abstract class RestTestBase {
 
-    private static final String URL = System.getProperty("kie.wb.url", "http://localhost:8080/kie-wb");
+    protected static final String URL = System.getProperty("kie.wb.url", "http://localhost:8080/kie-wb");
     protected static final String USER_ID = System.getProperty("kie.wb.user.name", User.REST_ALL.getUserName());
-    private static final String PASSWORD = System.getProperty("kie.wb.user.password", User.REST_ALL.getPassword());
+    protected static final String PASSWORD = System.getProperty("kie.wb.user.password", User.REST_ALL.getPassword());
+
+    protected static final String GIT_URL_REMOTE = "https://github.com/droolsjbpm/jbpm-playground.git";
```

Comment:
My concern with this is that the repository may change at any time and the tests could then start failing (and this can be really to figure out). It could also happen that github is down, it which case the test would fail as well. And also there is an issue with productized builds where the build actually does not have access to 3rd party sites, so the tests would likely fail.

Could we create a git repository as a resource on filesystem and use that? I am not saying we need to do it now, just trying to figure if there is a better way (even if implemented in future).